common/service/config/config.go

```diff
@@ -57,6 +57,9 @@ type (
 		ElasticSearch elasticsearch.Config `yaml:"elasticsearch"`
 		// PublicClient is config for connecting to cadence frontend
 		PublicClient PublicClient `yaml:"publicClient"`
+		// DCClientConfig is the config for setting up the file based dynamic config client
+		// Filepath should be relative to the root directory
+		DCClientConfig dynamicconfig.FileBasedClientConfig `yaml:"dcClientConfig"`
```

Comment:
Let's rename to "DynamicConfigClient", 
dc may not sounds familiar to customers, and team may confuse it with data center.

Reply:
Update.